core/src/main/java/org/jdbi/v3/core/array/SqlArrayTypes.java

```diff
@@ -35,10 +36,24 @@
     public SqlArrayTypes() {
         argumentStrategy = SqlArrayArgumentStrategy.SQL_ARRAY;
 
+        register(boolean.class, "boolean");
+        register(Boolean.class, "boolean");
+        register(short.class, "smallint");
+        register(Short.class, "smallint");
+        register(int.class, "integer");
+        register(Integer.class, "integer");
+        register(long.class, "bigint");
+        register(Long.class, "bigint");
+        register(float.class, "float4");
+        register(Float.class, "float4");
+        register(double.class, "float8");
+        register(Double.class, "float8");
+        register(String.class, "varchar");
+        register(UUID.class, "uuid");
         register(new EnumSqlArrayTypeFactory());
     }
 
-    private SqlArrayTypes(SqlArrayTypes that) {
+    private SqlArrayTypes(final SqlArrayTypes that) {
```

Comment:
Is this something we want to gradually make our standard code style? If not, I'd rather avoid all this churn since it's of rather little value.